packages/fast-tooling-react/src/form/custom-controls/index.ts

```diff
@@ -10,10 +10,10 @@ import { ThemeControlProps } from "./control.theme.props";
 export {
     AlignControl,
     AlignControlProps,
```

Comment:
Still TODO?

Reply:
Yes, I didn't vet those custom controls in this PR.